src/index.js

```diff
@@ -4,6 +4,8 @@ import bodyParser from 'body-parser';
 import cors from 'cors';
 import swaggerUI from 'swagger-ui-express';
 import doc from '../doc.json';
+import routes from './routes';
+import '@babel/polyfill';
```

Comment:
Please could you explain the reason for line 8 `import '@babel/polyfill'` because I think babel-node, called from `dev-start` script, does just fine.

Perhaps you experienced `ReferenceError: regeneratorRuntime is not defined` error when you ran your tests. The solution is to require @babel/polyfill in the test script. The snippet below shows what I mean:
```
{
  ...
  "test": " ... --require @babel/polyfill  'src/**/**.test.js' ...",
  ...
}
```

This solves the issue.

Reply:
Ok then I would remove from my file then. Thanks